README.md

```diff
@@ -1,63 +1,225 @@
-
-- [x] Elegant, Swifty APIs for all the things you need to build a backend including REST Routing, Jobs & Schedulers, Database access, a QueryBuilder & ORM, and much more
-- [x] A CLI & libraries built around facilitating shared code between your server & iOS clients. Write less code & rapidly develop full stack features, end to end.
-- [x] Extensive documentation, guides and tutorials to keep you focused on building your project.
+<p align="center">
+<a href="https://swift.org"><img src="https://img.shields.io/badge/Swift-5.3-orange.svg" alt="Swift Version"></a>
+<a href="https://github.com/joshuawright11/alchemy/releases"><img src="https://img.shields.io/github/release/joshuawright11/alchemy.svg" alt="Latest Release"></a>
+<a href="https://github.com/joshuawright11/alchemy/blob/main/LICENSE"><img src="https://img.shields.io/github/license/joshuawright11/alchemy.svg" alt="License"></a>
+</p>
+
+## About Alchemy
+
+Alchemy is a Swift web framework for building the backend of your next mobile app. It makes your development experience...
+
+- **Swifty**. Concise, expressive APIs built with the best parts of Swift.
+- **Safe**. Swift is built for safety. Its typing, optionals, value semantics and error handling are leveraged throughout Alchemy to help protect you against thread safety issues, nil values and unexpected program state.
+- **Rapid**. Write less code & rapidly develop full stack features, end to end. The CLI & supporting libraries (Papyrus, Fusion) are built around facilitating shared code & providing type safety between your server & iOS clients.
+- **Easy**. With elegant syntax, 100% documentation, and guides touching on nearly every feature, Alchemy is designed to help you build backends faster, not get in the way.
+- **Simple**. Juggle less Xcode projects by keeping your full stack Swift code in a monorepo containing your iOS app, Alchemy Server & Shared code. The CLI will help you get started.
+
+## Code Samples
+Alchemy is built to be both swifty and easy to follow. There is tons of sample code in the [Guides](Documentation/0_GettingStarted.md) and [quickstart projects](Quickstart/) but here are a few examples.
+
+### Hello, World!
+
+```swift
+import Alchemy
+
+struct MyServerApp: Application {
+    // Inject the global app router
+    @Inject router: Router
+
+    func setup() {
+        self.router.on(.post, at: "/hello") { request in
+            "Hello, World!"
+        }
+    }
+}
+
+// main.swift
+Launch<MyServerApp>.main()
+```
+
+### Databases & Rune ORM
+Raw queries, a Query builder, and an ORM (Rune), are all provided for interacting with SQL databases.
+```swift
+import Alchemy
+
+// Setup the default database.
+DB.default = PostgresDatabase(
+    DatabaseConfig(
+        socket: .ip(host: "localhost", port: 5432),
+        database: "alchemy",
+        username: "admin",
+        password: "password"
+    )
+)
+
+// Create a model matching a table
+struct Todo: Model {
+    static var let tableName = "todos"
```

Comment:
Shouldn't this just be `static let tableName = "todos"`

Reply:
Yep 👍